documentation/src/main/asciidoc/topics/con_invalidation_caches.adoc

```diff
@@ -1,11 +1,19 @@
 [id='invalidation-caches_{context}']
 = Invalidation caches
 
-You can use {brandname} in invalidation mode to optimize systems that perform high volumes of read operations.
-A good example is to use invalidation to prevent lots of database writes when state changes occur.
+Invalidation cache mode in {brandname} is designed to optimize systems that perform high volumes of read operations.
+You can use invalidation mode to reduce the number of database writes when state changes occur.
 
-This cache mode only makes sense if you have another, permanent store for your data such as a database and are only using {brandname} as an optimization in a read-heavy system, to prevent hitting the database for every read.
-If a cache is configured for invalidation, every time data is changed in a cache, other caches in the cluster receive a message informing them that their data is now stale and should be removed from memory and from any local store.
+Invalidation cache mode is effective only when you have a permanent data store, such as a database, and are only using {brandname} as an optimization in a read-heavy system to prevent hitting the database for every read.
+If a cache is configured for invalidation, each time data changes in a cache, other caches in the cluster receive a message informing them that their data is now stale and should be removed from memory and from any local store.
+Using invalidation mode without a persistent store is impractical, as updated values need to be read from a shared store for consistency across nodes.
+
+[IMPORTANT]
+====
+Invalidation cache mode is deprecated for {brandname} remote deployments.
+Use invalidation cache mode with embedded caches only.
+// guidance on how to get rid of stale entries with client/server mode
```

Comment:
Eviction and expiration?

Reply:
not sure what your question is

Reply:
Normally invalidation is used with the update to the external storage so it invalidates at the same time. If the user wants they can configure eviction or expiration, but I would say it doesn't really have any value here as eviction and expiration are the same for any cache mode.

Reply:
Agree. Thank you